libqtile/group.py

```diff
@@ -46,6 +46,7 @@ def __init__(self, name, layout=None, label=None):
         self.label = name if label is None else label
         self.custom_layout = layout  # will be set on _configure
         self.windows = []
+        self.tiled_windows = []
```

Comment:
Looks like we don't need to preserve order of this list, so let's make it a `set` so that duplicate items are impossible.

Reply:
yep!